core/arch/arm/plat-rockchip/conf.mk

```diff
@@ -56,6 +56,29 @@ CFG_SHMEM_SIZE   ?= 0x00400000
 CFG_EARLY_CONSOLE ?= n
 endif
 
+ifeq ($(PLATFORM_FLAVOR),rk3566)
+include core/arch/arm/cpu/cortex-armv8-0.mk
+arm32-platform-cpuarch  := cortex-a55
+
+$(call force,CFG_TEE_CORE_NB_CORE,4)
+# cortex-a55 sopports GICv3 or GICv4. There is no support for GICv4 in OP-TEE,
```

Comment:
What does the soc have?

Reply:
According to [rk3568 TRF](https://opensource.rock-chips.com/images/2/26/Rockchip_RK3568_TRM_Part1_V1.3-20220930P.PDF) the SoC has GIC-600, which [implements GICv3](https://developer.arm.com/Processors/CoreLink%20GIC-600).